### PR TITLE
fix(DB/creature): Add waypoint for Morin Cloudstalker

### PR DESCRIPTION
**Pull request by aleigood**
Adding a waypoint for the NPC Morin Cloudstalker

## CHANGES PROPOSED..:
- DATABASE

## ISSUES ADDRESSED:
Morin Cloudstalker is currently missing a waypoint like official data.

## TESTS PERFORMED:
Tested on a Linux server, NPC can walk with the waypoint.

## HOW TO TEST THE CHANGES:
.go creature id 2988 transmits to NPC location and observes its route of movement.

## KNOWN ISSUES AND TODO LIST:
None

## Target branch(es):
- [x] Master

## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by Si1ker** [comment]
@aleigood Thanks for your contribution, however, please follow the guideline to make a full PR next time at:
http://www.azerothcore.org/wiki/How-to-create-a-PR

Also, use the standard template when creating the PR:
https://github.com/azerothcore/azerothcore-wotlk/blob/master/pull_request_template.md

File: data/sql/updates/pending_db_world/rev_1585892474413237800.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+DELETE FROM `waypoint_data` WHERE `id`=@PATH;
+INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES (@PATH,1,-2369.16,-876.936,-9.42208,0,0,0,0,100,0),(@PATH,2,-2366.53,-856.691,-9.36085,0,0,0,0,100,0),(@PATH,3,-2357.15,-811.864,-9.42138,0,0,0,0,100,0),(@PATH,4,-2345.23,-748.967,-9.42454,0,0,0,0,100,0),(@PATH,5,-2341.13,-700.582,-9.29519,0,0,0,0,100,0),(@PATH,6,-2335.57,-673.555,-9.41523,0,0,0,0,100,0),(@PATH,7,-2323.8,-641.711,-9.34184,0,0,0,0,100,0),(@PATH,8,-2310.09,-613.217,-9.42431,0,0,0,0,100,0),(@PATH,9,-2295.38,-586.808,-9.4231,2.39676,21600,0,0,100,0),(@PATH,10,-2323.09,-595.375,-9.42242,0,0,0,0,100,0),(@PATH,11,-2312.27,-614.66,-9.42405,0,0,0,0,100,0),(@PATH,12,-2326.41,-645.536,-9.34993,0,0,0,0,100,0),(@PATH,13,-2336.51,-676.214,-9.40864,0,0,0,0,100,0),(@PATH,14,-2344.05,-723.843,-9.42353,0,0,0,0,100,0),(@PATH,15,-2346.75,-760.367,-9.42353,0,0,0,0,100,0),(@PATH,16,-2357.56,-811.652,-9.42353,0,0,0,0,100,0),(@PATH,17,-2366.12,-855.675,-9.32695,0,0,0,0,100,0),(@PATH,18,-2376.83,-896.428,-9.39643,6.2165,21600,0,0,100,0);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1585892474413237800.sql:9]: I suggest this for easier readability
```suggestion
INSERT INTO `waypoint_data` (`id`,`point`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`move_type`,`action`,`action_chance`,`wpguid`) VALUES 
(@PATH,1,-2369.16,-876.936,-9.42208,0,0,0,0,100,0),
(@PATH,2,-2366.53,-856.691,-9.36085,0,0,0,0,100,0),
(@PATH,3,-2357.15,-811.864,-9.42138,0,0,0,0,100,0),
(@PATH,4,-2345.23,-748.967,-9.42454,0,0,0,0,100,0),
(@PATH,5,-2341.13,-700.582,-9.29519,0,0,0,0,100,0),
(@PATH,6,-2335.57,-673.555,-9.41523,0,0,0,0,100,0),
(@PATH,7,-2323.8,-641.711,-9.34184,0,0,0,0,100,0),
(@PATH,8,-2310.09,-613.217,-9.42431,0,0,0,0,100,0),
(@PATH,9,-2295.38,-586.808,-9.4231,2.39676,21600,0,0,100,0),
(@PATH,10,-2323.09,-595.375,-9.42242,0,0,0,0,100,0),
(@PATH,11,-2312.27,-614.66,-9.42405,0,0,0,0,100,0),
(@PATH,12,-2326.41,-645.536,-9.34993,0,0,0,0,100,0),
(@PATH,13,-2336.51,-676.214,-9.40864,0,0,0,0,100,0),
(@PATH,14,-2344.05,-723.843,-9.42353,0,0,0,0,100,0),
(@PATH,15,-2346.75,-760.367,-9.42353,0,0,0,0,100,0),
(@PATH,16,-2357.56,-811.652,-9.42353,0,0,0,0,100,0),
(@PATH,17,-2366.12,-855.675,-9.32695,0,0,0,0,100,0),
(@PATH,18,-2376.83,-896.428,-9.39643,6.2165,21600,0,0,100,0);
```

Comment (aleigood) [reply data/sql/updates/pending_db_world/rev_1585892474413237800.sql:9]: I will improve the content of the template, you know, it is more difficult than writing code in non-English speaking countries.😂

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1585892474413237800.sql:9]: :P

**Comment by aleigood** [comment]
Hopefully the SQL_REV values won't conflict again.😥

**Comment by BarbzYHOOL** [comment]
that's weird, @FrancescoBorzi can you check what the hell is happening?? https://travis-ci.org/github/azerothcore/azerothcore-wotlk/builds/670580477?utm_source=github_status&utm_medium=notification only one travis failed and it's always the same error on each of aleigood's PR

**Comment by Si1ker** [comment]
2020-04-03 09_11_41-Window
From #2835  @BarbzYHOOL

**Comment by FrancescoBorzi** [comment]
Guys, what it's happening it's quite simple and it's nothing sophisticated nor strictly related to Travis.

The bash script that installs the DB is giving the following error when trying to import the SQL code:

> ERROR 1062 (23000) at line 77682: Duplicate entry '1585892474413237800' for key 'PRIMARY'

This means there is already a row having its ID (`sql_rev`) as `1585892474413237800` in the `version_db_world`.

We need to understand why this is happening. I don't have such an ID on a clean, latest-master DB

> SELECT * FROM version_db_world WHERE sql_rev = '1585892474413237800'

**Comment by BarbzYHOOL** [comment]
@aleigood I updated your branch slightly to try a fix, we can remove my commit later (or keep it, it won't matter), let's wait for travis to run again

**Comment by FrancescoBorzi** [comment]
@BarbzYHOOL cool, build passed

**Comment by Si1ker** [comment]
All working here ! ✔️ 
Ready to be merged 🚀 

2020-04-04 09_24_25-Window

**Comment by FrancescoBorzi** [comment]
well done @aleigood @BarbzYHOOL @Si1ker

**Comment by BarbzYHOOL** [comment]
lol that's not proper -_-'

i needed to make a separate bash PR but well it's fine i'll change it afterwards

**Comment by FrancescoBorzi** [comment]
@BarbzYHOOL I didn't know that, I thought that fix was final. Sorry

**Comment by BarbzYHOOL** [comment]
even if final, it's not proper, because this is not related to the PR and commit, I just pushed it here for quick testing :P but well, it's fine

**Pull request merged**